courses/machine_learning/images/mnist_models.ipynb

```diff
@@ -0,0 +1,1012 @@
+{
```

Comment:
<p>Is that working? </p><p> </p><p>(I would expect that for  "python3 -m trainer.task"  to work you'd need to pip install /trainer or add /trainer in the PYTHONPATH before running the command - </p><p>python3 /trainer/task.py should work though)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Yessir, works like a charm :) Next time we videochat, I'll show you. Or better yet, I'll figure out this weird error you're getting so you can see it for yourself.</p>